src/Microsoft.AspNetCore.SignalR/GetHttpContextExtensions.cs

```diff
@@ -7,8 +7,16 @@
 
 namespace Microsoft.AspNetCore.SignalR
 {
+    /// <summary>
+    /// Extension methods for accessing <see cref="HttpContext"/> from a hub context.
+    /// </summary>
     public static class GetHttpContextExtensions
     {
+        /// <summary>
+        /// Gets <see cref="HttpContext"/> from the specified connection.
+        /// </summary>
+        /// <param name="connection">The connection.</param>
+        /// <returns>The connection's <see cref="HttpContext"/>.</returns>
```

Comment:
"The [HttpContext] for the connection, or [null] if the connection is not associated with an HTTP request"